### PR TITLE
Refactor/move init state to app start

### LINKED ISSUE #170: initState on app start only

### PR DESCRIPTION
**Pull request by TerrenceLJones**
Resolves #170

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -135,44 +134,46 @@ const App = Marionette.Application.extend({
...
-    this.triggerStart(options);
+    this.triggerStart(opts);
 
     return this;
   },
 
   /**
-   * Triggers start event.
-   * Override to introduce async start
+   * Extends passed in state.
```
Comment (paulfalgout) [inline src/app.js]: I think it just gets the state, and not necessarily "extends"

**Comment by paulfalgout** [comment]
Trivial doc change and I think this is GTG

File: src/app.js (hunk 1 above)
Comment (TerrenceLJones) [inline src/app.js]: @paulfalgout i guess this needs to be update as well?
screen shot 2016-09-12 at 1 12 46 pm

Comment (paulfalgout) [inline src/app.js]: sure just state?

Comment (TerrenceLJones) [inline src/app.js]: That sentence didn't really connect. Come again?

Comment (TerrenceLJones) [inline src/app.js]: _Sigh_ @paulfalgout I just realized that the screenshot was the same location as your comment...I thought it was in the docs...but I just realized that I need to add docs on this function.

**Pull request merged**